test/DropdownButton.spec.js

```diff
@@ -1,28 +1,35 @@
 import React from 'react';
 import { shallow } from 'enzyme';
 import Dropdown from '../src/Dropdown';
-import mocker from './helper/mocker';
+import Divider from '../src/Divider';
+import mocker from './helper/new-mocker';
 
 describe('<Dropdown />', () => {
   let wrapper;
-  const dropdownMock = jest.fn();
-  const restore = mocker('dropdown', dropdownMock);
-
-  afterAll(() => {
-    restore();
-  });
 
   test('renders', () => {
     wrapper = shallow(<Dropdown trigger={<button>Drop me!</button>} />);
     expect(wrapper).toMatchSnapshot();
   });
 
-  test('handles options prop', () => {
-    const options = { constrainWidth: true };
+  test('does not update ID reference with each render', () => {
+    wrapper = shallow(<Dropdown trigger={<span>hi</span>} />);
+    const { idx } = wrapper.instance();
+    wrapper.render();
+    expect(wrapper.instance().idx).toEqual(idx);
+  });
+
+  test('does correctly render dividers', () => {
     wrapper = shallow(
-      <Dropdown trigger={<button>Drop me!</button>} options={options} />
+      <Dropdown trigger={<span>hi</span>}>
+        <a href="#">test</a>
+        <Divider />
+        <a href="#">test</a>
+      </Dropdown>
     );
-    expect(dropdownMock).toHaveBeenCalledWith(options);
+    const { idx } = wrapper.instance();
```

Comment:
what this line for?

Reply:
Mhh, apparently it does nothing.
Sorry for that, gotta remove it ASAP